script.js

```diff
@@ -0,0 +1,815 @@
+dealButton.innerHTML = "DEAL";
+dealButton.classList.add("disappear");
+
+//swap button
+const swapButton = document.createElement("button");
+swapButton.innerHTML = "SWAP";
+swapButton.classList.add("disappear");
+
+//calculate score button
+const calScore = document.createElement("button");
+calScore.innerHTML = "GET SCORE";
+calScore.classList.add("disappear");
+
+//reset game button
+const resetButton = document.createElement("button");
+resetButton.innerHTML = "NEXT ROUND";
+resetButton.classList.add("disappear");
+
+//container for the entire results board
+const resultsContainer = document.createElement("div");
+resultsContainer.classList.add("resultsContainer");
+
+//result board that display the results
+const resultDiv = document.createElement("div");
+resultDiv.classList.add("resultsBoard");
+
+//point systems
+const currentPoints = document.createElement("p");
+currentPoints.innerHTML = `Current points: ${playerPoints}`;
+currentPoints.classList.add("currentPts");
+const pointsbet = document.createElement("p");
+pointsbet.innerHTML = `Bet for current round:`;
+pointsbet.classList.add("betPts");
+resultDiv.appendChild(currentPoints);
+resultDiv.appendChild(pointsbet);
+
+resultsContainer.appendChild(resultDiv);
+
+//message board the shows prompt message to player
+const messageContainer = document.createElement("div");
+messageContainer.classList.add("container");
+const messageBoard = document.createElement("div");
+messageBoard.classList.add("msgBoardDiv");
+const promptMessage = document.createElement("p");
+promptMessage.innerHTML =
+  "Welcome to the SUPER FUN Poker Game!! Please start by placing a bet.";
+promptMessage.classList.add("promptMsg");
+messageBoard.appendChild(promptMessage);
+messageContainer.appendChild(messageBoard);
+
+//scoring system that shows all the multipliers for each combination
+const scoreSystem = document.createElement("div");
+scoreSystem.classList.add("scoreSys");
+scoreSystem.innerHTML = "Scoring System";
+
+const scoreTableDiv = document.createElement("div");
+scoreTableDiv.classList.add("disappear");
+
+const scoreTable = document.createElement("table");
+scoreTable.classList.add("fullScore");
+
+const scoreText = [
+  ["Combinations", "Mutliplier"],
+  ["A Pair of Jacks or Better", "5"],
+  ["Two Pairs ", "10"],
+  ["Three of a kind", "20"],
+  ["Straight", "40"],
+  ["Flush", "50"],
+  ["Full House", "100"],
+  ["Four of a Kind", "200"],
+  ["Straight Flush", "300"],
+];
+
+for (let i = 0; i < 9; i += 1) {
+  const row = document.createElement("tr");
+  for (let j = 0; j < 2; j += 1) {
+    const column = document.createElement("th");
+    column.innerText = scoreText[i][j];
+    row.appendChild(column);
+  }
+  scoreTable.appendChild(row);
+}
+
+scoreTableDiv.appendChild(scoreTable);
+scoreSystem.addEventListener("mouseover", () => {
+  scoreTableDiv.classList.remove("disappear");
+});
+scoreSystem.addEventListener("mouseleave", () => {
+  scoreTableDiv.classList.add("disappear");
+});
```

Comment:
consider putting these in helper functions so that your code is more organised and easier to read